tests/Containerfile-build

```diff
@@ -0,0 +1,11 @@
+FROM quay.io/centos/centos:stream9
+
+ARG USERNAME=test-user
+
+RUN dnf install -y dnf-plugin-config-manager
```

Comment:
Can you combine all of your run lines onto a single line, to prevent propagation of layers, also add a dnf -y clean all to the last line to remove execess noice in the image.  This will help shrink the size of the image.

Reply:
Thanks, forgot about the the layers, Ack